XenAdmin/Wizards/PatchingWizard/AutomatedUpdatesBasePage.cs

```diff
@@ -265,60 +268,60 @@ private void UpdateStatus()
                 var errorSb = new StringBuilder();
 
                 if (!string.IsNullOrEmpty(bgw.Name))
-                    sb.AppendLine($"{bgw.Name}:");
+                    sb.AppendFormattedLine($"{bgw.Name}:", addTimestamp);
 
                 foreach (var pa in bgw.DoneActions)
                 {
-                    pa.ProgressHistory.ForEach(step => sb.AppendIndented(step).AppendLine());
+                    pa.ProgressHistory.ForEach(step => sb.AppendFormattedLine(step, addTimestamp, true));

```

Comment:
Style remark: since there are two optional bool parameters, it would be helpful to add the label to the one used so there is no ambiguity. Same in the other lines where this method is called with the one of the optional parameters set).